README.md

```diff
@@ -0,0 +1,19 @@
+# Flawless
+
+Hello :wave:, we are team AbstractSingletonProxyFactoryBean,
+and this is our submission for [PyWeek 31](https://pyweek.org/)
+
+### Installation
+In your terminal of choice, run
+```commandline
+$ git clone https://github.com/SavagePastaMan/pyweek-2021.git
+$ cd directory_name
+$ python -m pip install -r requirements.txt
+$ python -m game
+```
+Depending on your platform, you may need to use `python3` or `py`.
+
+### Background
+As part of the biggest heist of the century, your goal is to steal a diamond. To do this, you will have to bypass many levels of security. As you get closer to the diamond, the security will become tougher and you will face new challenges. 
+
+More information can be found by selecting `How to Play` from the main menu in the game.
```

Comment:
Should we remove “in the game”?

Reply:
probs makes sense to remove it because it makes sense in context.